dp/core/ethdev.c

```diff
@@ -27,12 +27,50 @@
 
 #include <string.h>
 
+/* For struct sockaddr */
```

Comment:
I am not sure that you should merge the former ixgbe.c into ethdev.c. ethdev.c is one layer above ixgbe.c and should remain like that. It is more clean if you introduce a driver.c which contains code that is shared and common between different device drivers.